Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -344,6 +344,34 @@ def search_incidents(client, args):
     return response
 
 
+def extract_data_form_other_fields_argument(other_fields: str, incident, changes):
+    """Example function with types documented in the docstring.
+
+    Args:
+        other_fields: Contains the field that should change and the new value ({"name": {"text": "The new name"}}).
+        incident (dict): Contains the old value of the field that should change ({"name": "The old name"}).
+        changes (list): Contains the fields that should change with the old and new values in IBM format
+            ([{'field': {'name': 'confirmed'}, 'old_value': {'boolean': 'false'}, 'new_value': {'boolean': 'true'},
+            {'field': {'name': 'name'}, 'old_value': {'text': 'The old name'}, 'new_value': {'text': 'The new name'}}]).
+
+    """
+
+    try:
+        other_fields_json = json.loads(other_fields)
+    except Exception as e:
+        raise e
```

Comment:
I would add to the error message that the other_fields argument is not a valid json.